### ISSUE TITLE
Content Audit: Page-level analysis

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds-site#2409]
We have outlined the following goals for this content audit of the [USWDS website](https://designsystem.digital.gov/):
- Gather basic descriptive and performance data for all pages
- Combine with top tasks results
- Identify obsolete and redundant content
- Gauge and report on site’s adherence to web content best practices
- Gauge and report on site’s adherence to accessibility best practices
- Use data to inform IA and content improvements

[Documents related to the content audit](https://drive.google.com/drive/folders/1XU8xm1nUGG0VU1ogPZUfXkKQdXsbZss_?usp=drive_link) of the USWDS website are located in the project Google Drive.

In order to consider page-level analysis complete, the following tasks should be done:
- [x] All website pages have been reviewed and evaluated for their relationship with Top Tasks
- [ ] We have a list of obsolete and redundant content
- [ ] The team has been informed of the results of analysis and the site's adherence to web content and accessibility best practices
- [ ] The team has aligned on a plan for IA and content improvements

**Comment by sarah-sch** [comment]
Page-level review is complete. @brunerae and Sarah will work together to schedule a readout session for the team after the January monthly call.
